src/spikeinterface/qualitymetrics/tests/test_metrics_functions.py

```diff
@@ -570,27 +540,36 @@ def test_calculate_sd_ratio(sorting_analyzer_simple):
 
 if __name__ == "__main__":
 
-    sorting_analyzer = _sorting_analyzer_simple()
-    print(sorting_analyzer)
-
-    test_unit_structure_in_output(_small_sorting_analyzer())
-
-    # test_calculate_firing_rate_num_spikes(sorting_analyzer)
```

Comment:
I'll delete them and check the gh action test times. Thanks for the investigation!

Reply:
Yup - it made no difference. So... why was Windows taking so long before this PR?!

Reply:
🤔 very strange!! could it be the change in `n_jobs` number?

Reply:
Hi.
theses `if __name__ == "__main__":`are for me!
I am very very old fashion/lazy and my workflow for debugging is running flat python on test files the main is needed.
Do not destroy then too much I recreate then any time I need then. You can comment then if you want.

Reply:
Haha @samuelgarcia, the problem with commenting is they are super-confusing unless you know what they are for. I think the two downsides are:
- It can take up quite a lot of time trying to figure out what is going on with them (i.e. as above)
- leads to developers acting under the impression tests are being run that are actually not.
mais j'ai un proposition! can we fix an empty commit to main branch like `store test debugging conditionals`? Then a reference to these will exist forever that you can copy and paste from github whenever needed during debugging, but also, they can be deleted in future to avoid confusion. WDYT?

Reply:
The other option is to leave them there and above them just add a comment saying (testing for Sam only/local testing without pytest). Then things are documented so new devs don't get confused and Sam stays happy with his old-school style.

Reply:
If we're not removing them, I'd vote to comment them out and add a comment?

Reply:
I think at the meeting we were leaning towards adding an explanatory comment here. After some further consideration, I would say that leaving the debug code there and adding a comment would still make me very sad 😅 for the following three reasons:

1) It is against the main purpose of code comments, which is to assist in understanding of the codebase. Commenting out unused / debug code like this makes the codebase less easy to understand and is in general considered bad practice, with the preferred method to save such things in the git history. There are many articles such as [this one](https://kentcdodds.com/blog/please-dont-commit-commented-out-code) that can be found related to 'code comments best practices'.
2) This is just one situation, but does this mean we have a general rule that we can comment out our debug code in the codebase for convenience? This doesn't seem like a good idea, the codebase will soon be very messy.
3) Just in terms of optics it looks unprofessional, and will degrade the general readers confidence in the codebase.

But this is not really related to this PR,  maybe we can create a new issue / add to Friday's agenda! and then I will shut up about it 😅

Reply:
They don't even need to be commented out no? We just add the comment on top saying these are for people that prefer python/ipython testing. I've learned that it is best to pick the battles and if Sam and Alessio like it I would just leave them for now. Open an issue and then we try to move them in a new direction (so I'm flipping your default @JoeZiminski--ie leave them and then discuss to remove them later). Heberto has convinced me that somethings are nice just from a developer convenience perspective and since we are the current core of contributors we should help make things convenient. As far as other people reading the code base I would say at least half of users (I would posit more) are pure scientists and don't worry about the testing framework.

Reply:
Happy to revert the deletion for now as unrelated to the main trust of the PR. I think they should be at least commented (here or in a follow up PR), otherwise it really seems like they are being used (even with a clarifying comment at the top). But I agree this isn't the place to delete them and we can continue the discussion in a new issue.

Reply:
Cool - I've reverted and we can discuss further, later.